docs/user/config/diagnostics.md

```diff
@@ -2,18 +2,16 @@
 title: Diagnostics option
 ---
 
-The `diagnostics` option allows to enable/disable the error reporting. But it can also be used to filter which one should throw during tests, as well from which file it should be reported or not.
+The `diagnostics` option allows to configure error reporting. 
+It can both be enabled/disabled entirely or limited to a specific type of errors or files.
 
-A diagnostic can be:
-- an error in your TypeScript config (whether it's a file or inline)
-- syntax errors in some of your TypeScript files (source or tests)
-- type/semantic errors, what TypeScript has actually been made for 😁
```

Comment:
Why did this got removed? Was it in purpose?

Reply:
Yeah. I think for most users of the library the different types of diagnostics doesn't really matter (unless e.g. there was an ability to turn one category of them off)